modules/administration-guide/partials/assembly_configuring-prod-cli-to-use-a-backup-server-using-the-configuration-object.adoc

```diff
@@ -0,0 +1,18 @@
+:parent-context-of-configuring-prod-cli-to-use-a-backup-server-using-the-configuraiton-object: {context}
+
+[id="configuring-prod-cli-to-use-a-backup-server-using-the-configuraiton-object_{context}"]
+= Configuring {prod-cli} to use a backup server using the configuration object
+
+:context: configuring-prod-cli-to-use-a-backup-server-using-the-configuraiton-object
+
+You can configure {prod-cli} to use a backup server by pointing {prod-cli} to the backup server configuration object using a command argument or an environment variable.
```

Comment:
Here's a good example of how gerunds can bite you:
```suggestion
You can configure {prod-cli} to use a backup server by pointing {prod-cli} to the backup server configuration object by using a command argument or environment variable.
``` (I believe this is your intended meaning.)
```suggestion
You can configure {prod-cli} to use a backup server by pointing {prod-cli} to the backup server configuration object that uses a command argument or environment variable.
``` (This is a potential unintended meaning.)